CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+# Contribution Guidelines
+
+We use [uv](https://docs.astral.sh/uv/) for dependency management during development. UV is easily installed __without__
+the need for a virtual environment or `conda` environment, but one can install it inside a `conda` environment too.
+
+## Development model
+
+We use a fork-and-merge development model (see an introduction
+[here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks) if needed).
+
+The steps include:
+
+1.  Creating a personal fork of `ml4gw` repository in your personal namespace.
+2.  Create a branch in your fork and make changes on it.
+3.  Open a pull request.
+
+## Installation and proposing patches
+Once `uv` is installed, clone your _fork_ using
+```bash
+$ git clone https://github.com/<namespace>/ml4gw
```

Comment:
I think the `$` should be removed from the bash blocks - they get copied when clicking the "copy" button on GitHub.